### PR TITLE
MINOR: how to add spam control

### PR DESCRIPTION
**Pull request by sunnysideup**

### CODE REVIEW

**Comment by NightJar** [comment]
Suggest instead yaml config if module exists spamprotection commentcontroller.extensions - https://github.com/silverstripe/silverstripe-spamprotection/blob/4/code/Extension/CommentSpamProtection.php
& add composer.json suggests silverstripe/spamprotection

**Comment by GuySartorelli** [comment]
Can you please add some information describing what this will actually _do_?
Please also retarget this to the `3.10` branch.

File: docs/en/Configuration.md [hunk 1]
```diff
@@ -75,6 +75,20 @@ use SilverStripe\CMS\Model\SiteTree;
...
+you can create a comment.yml file with the following content:
+
+```yml
+---
+Name: 'app_comments_spam'
+---
+
+Page:
+  extensions:
+    - SilverStripe\SpamProtection\Extension\CommentSpamProtection
```
Comment (ChloeSartorelli) [inline docs/en/Configuration.md:89]: That extension seems to come from a separate module - please point developers to the relevant module instead, and to any documentation that module has for its usage.

**Comment by sunnysideup** [comment]
Lets do this:

> 
> Suggest instead yaml config if module exists spamprotection commentcontroller.extensions - https://github.com/silverstripe/silverstripe-spamprotection/blob/4/code/Extension/CommentSpamProtection.php
> & add composer.json suggests silverstripe/spamprotection

**Comment by GuySartorelli** [comment]
@sunnysideup Are you still interested in this PR? I will close it if there has still been no action on it after a week.

**Comment by GuySartorelli** [comment]
I'm closing this PR - feel free to reopen it if you decide to pick it up again.

**Pull request closed without merging**